app/controllers/events_controller.rb

```diff
@@ -0,0 +1,75 @@
+class EventsController < ApplicationController
+  before_action :set_event, only: [:show, :edit, :update, :destroy]
+  before_action :authorize_project_member
+  before_action :authorize_event_author, only: [:update, :destroy]
+
+  def index
+    @project = Project.find(params[:project_id])
```

Comment:
it's because in the view I need project id, I could pass it separately (`@events` and `@project_id`) but I pass the whole `@project` instead